File: app/src/main/java/com/example/msdc/api/CreditCastResult.java
package com.example.msdc.api;

import com.google.gson.annotations.SerializedName;

public class CreditCastResult {
    @SerializedName("id")
    int id;

    @SerializedName("credit_id")
    String creditId;

    @SerializedName("name")
    String name;

    @SerializedName("profile_path")
    String profilePath;

    @SerializedName("character")
    String character;

    @SerializedName("gender")
    int gender;

    public int getId() {
        return id;
    }

    public String getCreditId() {
        return creditId;
    }

    public String getName() {
        return name;
    }

    public String getProfilePath() {
        return profilePath;
    }

    public String getCharacter() {
        return character;
    }

    public int getGender() {
        return gender;
    }
}

File: app/src/main/java/com/example/msdc/api/CreditCastResult.kt
package com.example.msdc.api

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class CreditCastResult (
    @field:SerializedName("id") val id : Int? = 0,
    @field:SerializedName("name") val name: String? = null,
    @field:SerializedName("profile_path") val profilePath: String? = null,
    @field:SerializedName("character") val character: String? = null,
    @field:SerializedName("gender") val gender : Int? = 0
) : Parcelable